backend/items/management/commands/export_to_ndjson.py

```
@@ -0,0 +1,168 @@
+        
+        self.stdout.write(self.style.SUCCESS(f"\n{'='*60}"))
+        self.stdout.write(self.style.SUCCESS("Export Complete!"))
+        self.stdout.write(self.style.SUCCESS(f"{'='*60}"))
+        self.stdout.write(f"\nOutput directory: {output_dir.absolute()}\n")
+
+    def _export_users(self, output_dir, overwrite, verbose):
+        """Export users to users.ndjson"""
+        output_file = output_dir / 'users.ndjson'
+        
+        if output_file.exists() and not overwrite:
+            self.stdout.write(self.style.WARNING(f"Skipping users.ndjson (already exists, use --overwrite)"))
```

Ruff (0.15.2)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)